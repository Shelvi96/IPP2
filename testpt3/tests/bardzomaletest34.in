NEW ID

42:987;>39902
31005434>:8
21541273;>:2;6;8
55579>05180;4
294:2:55:1>4064949
:88891152>21
39>6833:43289
92044>445
53;:>4906
:6224:5::3>891459
971>2
5;9633>29;85158
0>8920
;8;>168
;;897;3>12964268
@44
@8;98
@71888;;89670706
@72;526162:61;64
@80;84:4593544313:4
@029;40963
@22061
@:0439705;8:266;
@80950734;5:
@;0;:475526222140;7:880
@0788:1
@10;427;8016230700772
@72
@0988;17
@16481345928023462489;3930
