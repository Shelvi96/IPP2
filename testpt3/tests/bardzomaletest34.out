0
0
1
65
7291
0
0
133
0
1120011000
0
4842018
0
0
1022112100000
